source/lambda/kendraDataSourceCreator/lambda_function.py

```diff
@@ -0,0 +1,41 @@
+import boto3
+import os
+
+def on_create(event, context):
+    print("creating an s3 data source and connector for the Covid dataset")
+    kendra_client = boto3.client('kendra')
+    kendraIndexId = os.environ['KENDRA_INDEX_ID']
+    print(kendraIndexId)
```

Comment:
Please use more descriptive log statements. Something on the lines: 'Creating data source for kendra_id_id'